source/includes/options-mongosqld.yaml

```diff
@@ -828,4 +828,26 @@ description: |
      +------+------+--------+-------+
 
 optional: true
+---
+program: mongosqld
+name: setParameter
+directive: option
+args: <polymorphic_type_conversion_mode | type_conversion_mode>
+description: |
+
+  .. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Corresponds to
+
+   * - ``polymorphic_type_conversion_mode``
```

Comment:
Change to 
```
:setting:`setParameter.polymorphic_type_conversion_mode`
```

Reply:
Per your request, I removed all of the content related to this pr from this file.